extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -21,10 +28,17 @@ <h4>REST Resources</h4>
  </div>
  {/if}
  {/for}
-</header>
-
-<header>
-
-</header>
+ <h5>Additional Endpoints</h5>
+ {#for additionalEndpoint in info:additionalEndpointInfo}
+ <div class="row">
+  <div class="col">
+   {#if additionalEndpoint.description}
+   {additionalEndpoint.endpoint} &emsp; <a href="{additionalEndpoint.endpoint}">{additionalEndpoint.description}</a>
```

Comment:
I don't think we need to add the `{additionalEndpoint.endpoint} &emsp;` part: the link is enough.

Reply:
Hmm that make sense :)